ndm-operator.yaml

```diff
@@ -42,7 +42,7 @@ data:
         name: path filter
         state: true
         include: ""
-        exclude: loop
+        exclude: "loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/md,/dev/rbd"
```

Comment:
why loop does not have "/dev" and others have?

Reply:
We are only check if the blockdevice devpath contains the string `loop` or `/dev/fd0`. So both will work. Will keep it as `/dev/loop` to avoid confusion